### DISCUSSION TITLE
Disable spellcheck for code

### DISCUSSION THREAD

**Discussion by XingSc29** [TriliumNext/Trilium#3132] [Ideas]
Is it possible to disable spellcheck for codes between `` like `'a'.constructor.prototype.charAt=[].join;`?

**Comment by zadam** [comment]
Hi, unfortunately it's probably not possible ATM.

**Comment by michaelkasede** [comment]
Please disable spellcheck by default and allow users to enable it when they need it.

**Reply by Marci107** [reply]
Yes, it would be a huge update. It would make easier to write non english texts.

**Reply by zadam** [reply]
You can easily disable it in the Options:

![image](https://github.com/zadam/trilium/assets/617641/b7dcf60c-6972-488b-8572-8aa1a96de3de)
